core/src/miner/miner.rs

```diff
@@ -412,50 +395,29 @@ impl Miner {
 
     /// Prepares work which has to be done to seal.
     fn prepare_work(&self, block: ClosedBlock, original_work_hash: Option<H256>) {
-        let (work, is_new) = {
-            let mut sealing_work = self.sealing_work.lock();
-            let last_work_hash = sealing_work.queue.peek_last_ref().map(|pb| pb.block().header().hash());
+        let mut sealing_work = self.sealing_work.lock();
```

Comment:
@sgkim126 What do you think about removing SealingWork and SealingQueue. IMO, they were used by PoW consensus to try to make multiple blocks' seal at the same time. When we are using the Tendermint Consensus, we only create a seal at a time.
Also in another point of view, the sealwing_work.queue was needed because sealing in the PoW consensus is an asynchronous job. Creating a seal in the Tendermint consensus is a blocking job. We do not need to use the queue.

Reply:
Looks good. I'll try.